src/spriteset_map.cpp

```diff
@@ -40,6 +41,13 @@ Spriteset_Map::Spriteset_Map() {
 		character_sprites.push_back(EASYRPG_MAKE_SHARED<Sprite_Character>(i->second.get()));
 	}
 
+	Game_Vehicle* vehicle;
+	for (int i = 1; i <= 3; ++i) {
+		vehicle = Game_Map::GetVehicle((Game_Vehicle::Type) i);
+		if (vehicle->IsInCurrentMap())
```

Comment:
That would be slightly slightly slower, because it would update those sprites each frame even if they're not visible. However, the performance difference would be minimal, but I don't see what harm is this check doing.